test/integration/examples-bad/map_rect/stanc.expected

```diff
@@ -15,7 +15,7 @@ Instead supplied arguments of incompatible type: (vector, vector, real[], int[])
 
   $ ../../../../../install/default/bin/stanc bad_data_int_const.stan
 
-Semantic error at file bad_data_int_const.stan, line 21, columns 8-104:
+Semantic error at file bad_data_int_const.stan, lines 21-22, column 8:
```

Comment:
I can't make sense of this error message.  Is that column 8 in both lines 21 and 22?

Reply:
So far, I've opted to print only the start column in case the error is spread over multiple lines. Should I print both the start and end column in that case as well?

The bigger question I have is what we should print if an error is spread out over multiple files.